jwt-react/src/components/App/App.js

```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import { UserContext } from '../../contexts/userContext.js';
+import './App.css';
+
+import Login from '../Login/Login.js';
+import AuthenticatedPingTest from '../AuthenticatedPingTest/AuthenticatedPingTest.js';
+
+
+function App() {
+  const [user, setUser] = useState(null);
+  const isUserLoggedIn = () => {
+    return !!user;
+  }
+
+  return (
+    <div className="App">
+      <UserContext.Provider value={{user, setUser, isUserLoggedIn}}>
```

Comment:
Although I agree with escaping this later in this pr, on I don't think that "user" should be escaped in this instance it's purely just passing "user" as an argument so that it can be used with the "useContext" hook of react (basically so that user can be used in child components). 

Hopefully that makes sense, I'm definitely open to change, but the above is something to consider. :smile:

Reply:
```suggestion
      // since the values are stored as variables in context, it's not necessary to escape the values for XSS-sake.
      <UserContext.Provider value={{user, setUser, isUserLoggedIn}}>
```

I see. That makes sense. I'm not totally aware of what `Context` means in React land, but I think I got the gist of it. Thanks for comparing this to the other part of the PR that I reviewed. I think it's reasonable to perhaps leave a suggestion as a comment as to why escaping is not necessary?

Reply:
```suggestion
      // since the values are stored as variables in context, it's not necessary to escape the values for XSS-sake.
      <UserContext.Provider value={{user, setUser, isUserLoggedIn}}>
```

I see. That makes sense. I'm not totally aware of what `Context` means in React land, but I think I got the gist of it. Thanks for comparing this to the other part of the PR that I reviewed. I think it's reasonable to perhaps leave a suggestion as a comment as to why escaping is not necessary?

Reply:
Per your other comment regarding JSX, would user, in this case, be automatically escaped?